### PR TITLE
reduce common directory

### PR DESCRIPTION
**Pull request by Leospard**
**What this PR does**: 
reduce common directory
**Which issue(s) this PR fixes**: 
Fixes # https://github.com/apache/dubbo-go/issues/1937

**You should pay attention to items below to ensure your pr passes our ci test**
We do not merge pr with ci tests failed

- [x] All ut passed (run 'go test ./...' in project root)
- [ ] After go-fmt ed , run 'go fmt project' using goland.
- [ ] Golangci-lint passed, run 'sudo golangci-lint run' in project root.
- [ ] After import formatted, (using [imports-formatter](https://github.com/dubbogo/tools#5-how-to-get-imports-formatter) to run 'imports-formatter .' in project root, to format your import blocks, mentioned in [CONTRIBUTING.md](https://github.com/apache/dubbo-go/blob/master/CONTRIBUTING.md) above) 
- [ ] Your new-created file needs to have [apache license](https://raw.githubusercontent.com/dubbogo/resources/master/tools/license/license.txt) at the top, like other existed file does.
- [ ] All integration test passed. You can run integration test locally (with docker env). Clone our [dubbo-go-samples](https://github.com/apache/dubbo-go-samples) project and replace the go.mod to your dubbo-go, and run 'sudo sh start_integration_test.sh' at root of samples project root. (M1 Slice is not Support)

### CODE REVIEW

File: cluster/cluster/adaptivesvc/cluster_invoker.go [hunk 1]
```diff
@@ -20,21 +20,17 @@ package adaptivesvc
...
-
-import (
-	perrors "github.com/pkg/errors"
-)
 
-import (
 	"dubbo.apache.org/dubbo-go/v3/cluster/cluster/base"
 	"dubbo.apache.org/dubbo-go/v3/cluster/directory"
 	"dubbo.apache.org/dubbo-go/v3/cluster/metrics"
+	perrors "github.com/pkg/errors"
```
Comment (AlexStocks) [inline cluster/cluster/adaptivesvc/cluster_invoker.go:27]: 不要把 import 顺序搞乱

Comment (AlexStocks) [reply cluster/cluster/adaptivesvc/cluster_invoker.go:27]: 使用 https://github.com/dubbogo/tools/tree/master/cmd/imports-formatter 这个工具 format 下

Comment (Leospard) [reply cluster/cluster/adaptivesvc/cluster_invoker.go:27]: 好的！

**Review by AlexStocks — APPROVED** [review]

File: xds/resolver/watch_service.go [hunk 2]
```diff
@@ -30,7 +30,10 @@ import (
 )
 
 import (
-	dubboLogger "dubbo.apache.org/dubbo-go/v3/common/logger"
+	dubboLogger "github.com/dubbogo/gost/log/logger"
```
Comment (AlexStocks) [inline xds/resolver/watch_service.go:33]: pla rename its name to 'dubbogoLogger'.  pls change all codes in other places.

Comment (Leospard) [reply xds/resolver/watch_service.go:33]: OK👌

**Comment by AlexStocks** [comment]
pls fix the confliction

**Pull request merged**